flink-formats/flink-avro-confluent-registry/src/main/java/org/apache/flink/formats/avro/registry/confluent/RegistryAvroOptions.java

```diff
@@ -18,12 +18,14 @@
 
 package org.apache.flink.formats.avro.registry.confluent;
 
+import org.apache.flink.annotation.PublicEvolving;
 import org.apache.flink.configuration.ConfigOption;
 import org.apache.flink.configuration.ConfigOptions;
 
 import java.util.Map;
 
 /** Options for Schema Registry Avro format. */
+@PublicEvolving
 public class RegistryAvroOptions {
```

Comment:
shall we always use the factory identifier? in this case `AvroConfluentOptions`?